README.md

```diff
@@ -154,7 +154,12 @@ end
 oneview_fc_network 'Fc1' do
   client <my_client>
   data <resource_data>
-  action [:create, :create_if_missing, :delete]
+  associated_san <san_name> # String - Optional. Can also set managedSanUri in data
+  operation <op>            # String. Used in patch action only. e.g., 'add'
+  path <path>               # String. Used in patch option only. e.g., '/scopeUris/-'
+  value <val>               # String. Used in patch option only. e.g., 'scope uri'
+  scopes [<scope_names>]    # Array - Optional. Array of scope names. Used in add_to_scopes, remove_from_scopes or replace_scopes options only. e.g., ['Scope1', 'Scope2']
```

Comment:
If you are saying the scope_names is an Array, you do not need to put it inside brackets